lib/defaults/defaults.go

```diff
@@ -100,6 +100,10 @@ const (
 	// By default all users use /bin/bash
 	DefaultShell = "/bin/bash"
 
+	// GRPCMaxConcurrentStreams is the max GRPC streams that can be active at a time.  Once the limit is reached new
+	// RPC calls will queue until capacity is available.
+	GRPCMaxConcurrentStreams = 1000
```

Comment:
While I am not a fan of these `defaults` and `constants` packages at all, I think this value is more of a constant than a default and should be moved.

Defaults are things that can be overridden. If there's not a way to change this value and a particular build of Teleport will always use 1000, then it's a constant.

Reply:
What's a better location for this constant?

Reply:
Actually, let's just leave it. I was thinking of `api/constants` when I wrote this, but it's not needed by the API client and would be awkward there.